### ISSUE TITLE
Not able to Run Create Model and Load Trained weights module

### ISSUE THREAD

**Issue by swatinair123** [matterport/Mask_RCNN#272]
Hi I am getting this following error as in the screen shot when i run ## Create Model and Load Trained Weights section  .Please kindly advise what could be the issue and how to resolve
![screenshot](https://user-images.githubusercontent.com/23516228/36522281-bc93ca22-17c1-11e8-9434-138f16f02c44.jpg)

OSError                                   Traceback (most recent call last)
<ipython-input-7-138183a2a830> in <module>()
      3 
      4 # Load weights trained on MS-COCO
----> 5 model.load_weights(COCO_MODEL_PATH, by_name=True)

~\MaskRCNN\Mask_RCNN\model.py in load_weights(self, filepath, by_name, exclude)
   2015         if h5py is None:
   2016             raise ImportError('`load_weights` requires h5py.')
-> 2017         f = h5py.File(filepath, mode='r')
   2018         if 'layer_names' not in f.attrs and 'model_weights' in f:
   2019             f = f['model_weights']

c:\users\swati\anaconda\envs\maskrcnn\lib\site-packages\h5py\_hl\files.py in __init__(self, name, mode, driver, libver, userblock_size, swmr, **kwds)
    267             with phil:
    268                 fapl = make_fapl(driver, libver, **kwds)
--> 269                 fid = make_fid(name, mode, userblock_size, fapl, swmr=swmr)
    270 
    271                 if swmr_support:

c:\users\swati\anaconda\envs\maskrcnn\lib\site-packages\h5py\_hl\files.py in make_fid(name, mode, userblock_size, fapl, fcpl, swmr)
     97         if swmr and swmr_support:
     98             flags |= h5f.ACC_SWMR_READ
---> 99         fid = h5f.open(name, flags, fapl=fapl)
    100     elif mode == 'r+':
    101         fid = h5f.open(name, h5f.ACC_RDWR, fapl=fapl)

h5py\_objects.pyx in h5py._objects.with_phil.wrapper()

h5py\_objects.pyx in h5py._objects.with_phil.wrapper()

h5py\h5f.pyx in h5py.h5f.open()

OSError: Unable to open file (truncated file: eof = 153075712, sblock->base_addr = 0, stored_eof = 257557808)

**Comment by alwansm** [comment]
did you donloaded the "mask_rcnn_coco.h5" and move it to Mask_RCNN_Master?

**Comment by swatinair123** [comment]
Yes , But i am not getting the above error but when i give  the donut image from the test images for Run object Detection its says 

NameError                                 Traceback (most recent call last)
<ipython-input-8-afaef4e7abaf> in <module>()
     10 r = results[0]
     11 visualize.display_instances(image, r['rois'], r['masks'], r['class_ids'], 
---> 12                             class_names, r['scores'])

NameError: name 'class_names' is not defined
Can you please help me with this

**Comment by alwansm** [comment]
if it's in the training section it should be "dataset_train.class_names" not "class_names"

**Comment by yiakwy** [comment]
I have the same problem each time I load coco pretrained model after I clean the data. (rm mask_rcnn_coco.h5). Is the pretrained model corrupted ?

**Comment by xiongshuai520** [comment]
I have the same problem and anybody answer it or handle it well?thanks a lot

**Comment by xiongshuai520** [comment]
reload the coco.h5 dataset and it well be fine，Make sure the size of the data set is about 250M

**Comment by Tun555** [comment]
i have problem anyone have been found?

from mrcnn.model import MaskRCNN
NameError: name 'MaskRCNN' is not defined

**Comment by parth-singh71** [comment]
Hey @Tun555 Maybe this might solve your issue, change directory to the root of your MaskRCNN project and then run `python setup.py install` in your terminal or cmd.
